libs/libs.go

```diff
@@ -0,0 +1,9 @@
+package libs
+
+import "github.com/rumlang/rum/runtime"
+
+//LoadStdLib the basic lib into Context
+func LoadStdLib(ctx *runtime.Context) {
```

Comment:
do not need this function if move strings and csv to pkgs and load in init function